src/hyperion/device_setup_plans/manipulate_sample.py

```diff
@@ -12,16 +16,24 @@
 
 
 def setup_sample_environment(
+    aperture_scatterguard: ApertureScatterguard,
+    aperture_position: SingleAperturePosition,
     detector_motion: DetectorMotion,
     backlight: Backlight,
     attenuator: Attenuator,
     transmission_fraction: float,
     detector_distance: float,
     group="setup_senv",
 ):
-    """Move out the backlight, retract the detector shutter, and set the attenuator to
-    transmission."""
+    """Move the aperture into the position determined during XRC, Move out the backlight, retract the detector shutter,
+    and set the attenuator to transmission."""
```

Comment:
Should: It is from XRC in this case but it won't be generally, I think just `move the aperture into required position` is fine